### DISCUSSION TITLE
Why is only one CPU core used all the time

### DISCUSSION THREAD

**Discussion by 13330863440** [madMAx43v3r/chia-plotter#349] [Q&A]
args
```
.\chia_plot.exe
-t C:\ChiaTemp\
-2 Z:\ChiaTemp2\
 -d C:\ChiaPlots\
-r 54
-u 256
-p xxx 
-f xxx
```
Hardware：
128G RAM . set 116G ramdisk fot temp2 . ssd for temp1
CPU E5 2696V3 (18core,36thread) *2

why only use one cpus(36 threads)
i try set -r 72   Still so

All CPUs cannot be used.

How do I make the CPU use 100% ?

add memory to 256G ?

**Comment by james-chia** [comment]
Good question, same issue or its probably designed to be like this

**Comment by ataa** [comment]
I have Dual E5-2620 v4, 64GB Ram, 32 thread and Single 4TB nvme, On CentOS 8.3, I can see 70-80% usage on all cores.

**Comment by Hardcore-fs** [comment]
Same old story.....
incomplete information.....
no os details, no details about other processes.....
